doc/documentation_workflow/user_documentation_workflow.rst

```diff
@@ -0,0 +1,130 @@
+
+We work with `GitHub <https://www.github.com>`_ as the web-based hosting service for Git. Git allows us to keep our versions under control, with each release of NEST having its own documentation.
+
+.. image:: ../_static/img/documentation_workflow.png
+  :width: 300
+  :alt: Alternative text
+
+
+.. note::
+   This workflow shows you how to create **user documentation** for NEST. For **developer documentation**, please refer to our :doc:`Developer documentation workflow <developer_documentation_workflow>`.
+
+Changing the documentation
+++++++++++++++++++++++++++
+
+If you notice any errors or weaknesses in the documentation, please submit an `issue <https://github.com/nest/nest-simulator/issues>`_ in our GitHub repository.
+
+You can also make changes directly to your forked copy of the `NEST source code repository <https://github.com/nest/nest-simulator>`_ and create a `pull request <https://github.com/nest/nest-simulator/pulls>`_. Just follow the workflow below!
+
+Setting up your environment
++++++++++++++++++++++++++++
+
+To keep things simple, we have created a conda environment for you. Installing it will enable you to smoothly generate documentation for NEST.
+
+If you are using Linux and want to install a full development environment:
+
+1. Install conda (we recommend `miniconda <https://docs.conda.io/en/latest/miniconda.html#>`_).
+
+2. Switch to the ``doc`` folder in the source directory:
+
+.. code-block:: bash
+
+    cd </path/to/nest_source>/doc
+
+3. Create and activate the environment:
+
+.. code-block:: bash
+
+   conda env create -f nest_doc_conda_env.yml
+   conda update -n base -c defaults conda
+   conda activate nest-doc
+
+4. If you want to deactivate or delete the build environment:
+
+.. code-block:: bash
+
+   conda deactivate
+   conda remove --name nest-doc --all
+
+Generating documentation with Sphinx
+++++++++++++++++++++++++++++++++++++
+
+Now that you activated your environment, you can generate HTML files using Sphinx.
+
+Sphinx uses reStructuredText. To learn more on how to use it, check out this `quick reference <https://docutils.sourceforge.io/docs/user/rst/quickref.html>`_.
+
+Editing and creating pages
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. You can edit and/or add ``.rst`` files in the ``doc`` directory using your editor of choice.
+
+2. If you create a new page, open ``contents.rst`` in the ``doc`` directory and add the file name under ``.. toctree::``. This will ensure it appears on the NEST simulator documentation's table of contents.
+
+3. If you rename or move a file, please make sure you update all the corresponding cross-references.
+
+4. Save your changes.
+
+Rendering HTML
+~~~~~~~~~~~~~~
+
+Using Sphinx, you can build documentation locally and preview it offline:
+
+1. Go to the ``doc`` folder in the source directory:
+
+.. code-block:: bash
+
+    cd </path/to/nest_source>/doc
+
+2. Generate HTML files:
+
+.. code-block:: bash
+
+   make html
```

Comment:
The environment does not contain the required breathe extension
```
Extension error:
Could not import extension breathe (exception: No module named 'breathe')
Makefile:20: recipe for target 'html' failed
```

Reply:
See comment above